spec/models/orchestration_template_spec.rb

```diff
@@ -292,6 +292,13 @@
     end
   end
 
+  # Temporarily maintain backward compatibility for API users. Should be removed later.
+  describe '.new' do
+    it 'creates new type from old type' do
+      expect(described_class.new(:type => 'OrchestrationTemplateCfn')).to be_a(ManageIQ::Providers::Amazon::CloudManager::OrchestrationTemplate)
+    end
+  end
+
```

Comment:
@bzwei do you mind adding a test with .new("type" => ....) as inbound requests from API have string attributes and not symbols.  Thanks.